docs/functions/data.rst

```diff
@@ -95,6 +95,16 @@ Development Environment
 
    .. _`Red Hat Software Collection`: https://developers.redhat.com/products/softwarecollections/overview
 
+.. function:: _lp_terraform_env() -> var:lp_terraform_env
+
+   Returns ``true`` if a Terraform workspace is detected. Returns the workspace
+   name.
+
+   Can be disabled by :attr:`LP_ENABLE_TERRAFORM`.
```

Comment:
This should say enabled instead, since the default is disabled.

```suggestion
   Can be enabled by :attr:`LP_ENABLE_TERRAFORM`.
```


However, this feature could be enabled by default, since the check is only run if Terraform is installed. I'll let you decide what the default should be, I'm fine either way.